tools/jdbc/src/main/java/org/duckdb/DuckDBPreparedStatement.java

```diff
@@ -445,17 +453,29 @@ public int getResultSetType() throws SQLException {
 
     @Override
     public void addBatch(String sql) throws SQLException {
```

Comment:
Can you add a test for this method/include it in an existing test please?

Reply:
Definitely!